app/models/docker_container_wizard_states/environment.rb

```diff
@@ -12,7 +12,13 @@ class Environment < ActiveRecord::Base
              :validate => false
     include ::ParameterValidators
 
+    has_many :exposed_ports,  :dependent  => :destroy, :foreign_key => :reference_id,
+                              :inverse_of => :environment,
+                              :class_name => 'DockerContainerWizardStates::ExposedPort',
+                              :validate => false
+
     accepts_nested_attributes_for :environment_variables, :allow_destroy => true
+    accepts_nested_attributes_for :exposed_ports, :allow_destroy => true
```

Comment:
This is currently accepting words, as the docker API does accept them.. can we validate this to only accept numbers within 1-65535 ?